modules/core/include/opencv2/core/types.hpp

```diff
@@ -886,20 +889,65 @@ class CV_EXPORTS TermCriteria
 
     //! default constructor
     TermCriteria();
+
     /**
-    @param type The type of termination criteria, one of TermCriteria::Type
-    @param maxCount The maximum number of iterations or elements to compute.
-    @param epsilon The desired accuracy or change in parameters at which the iterative algorithm stops.
-    */
+     * @deprecated
+     * @param type The type of termination criteria, one of TermCriteria::Type
+     * @param maxCount The maximum number of iterations or elements to compute.
+     * @param epsilon The desired accuracy or change in parameters at which the iterative algorithm
+     * stops.
+     */
     TermCriteria(int type, int maxCount, double epsilon);
 
+    /**
+     * @param count
```

Comment:
Add parameters information describing constructed object.
E.g. in case of `count < 0` and `_epsilon < 0`.